qiskit/transpiler/passes/optimization/crosstalk_adaptive_schedule.py

```diff
@@ -719,8 +722,8 @@ def run(self, dag):
         """
         self.dag = dag
 
+        self.qubit_indices = {bit: dag.find_bit(bit).index for bit in dag.qubits}
```

Comment:
I was starting to do a second level review, because I believe there are some instances where these `qubit_indices` can be removed, and `find_bit` used further down in the module. I'll report here when I've gone over all of those.